packages/crayons-core/src/components/icon/icon.tsx

```diff
@@ -57,94 +50,31 @@ export class Icon {
    * Color in which the icon is displayed, specified as a standard CSS color or as a HEX code.
    */
   @Prop() color = '';
-  /**
-   * Enable Intersection Observer. Default is false.
-   */
-  @Prop() lazy = false;
   /**
    * Name of External Library to be used
    */
   @Prop() library = 'crayons';
+  /**
+   * Enable Intersection Observer. Default is false.
+   */
+  @Prop() lazy = false;
 
-  @Element() el: HTMLElement;
+  @State() private setElVisible = false;
```

Comment:
Why is this a State? can't it just be a private variable?

Reply:
@parsuram-vijaysankar The @State() decorator can be used to manage internal data for a component. This means that a user cannot modify this data from outside the component, but the component can modify it however it sees fit. Any changes to a @State() property will cause the components render function to be called again.
I have just refactored the earlier implementation with proper variable and function names and retested exception cases, and removed unnecessary exception logging.